packages/elements/bolt-type/type.schema.js

```diff
@@ -0,0 +1,56 @@
+module.exports = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'Type',
+  type: 'object',
+  properties: {
+    attributes: {
+      type: 'object',
+      description:
+        'A Drupal-style attributes object with extra attributes to append to this element.',
+    },
+    content: {
+      type: 'string',
```

Comment:
Same idea as my other comment, this should probably be
```suggestion
      type: 'any',
```